### PR TITLE
Refactor UncommunicativeMethodName.

### PR DESCRIPTION
**Pull request by troessner**
Follow-up to #691 

Adressed all comments - please re-review.

### CODE REVIEW

File: lib/reek/smells/uncommunicative_module_name.rb [hunk 1]
```diff
@@ -21,13 +21,13 @@ class UncommunicativeModuleName < SmellDetector
...
       REJECT_KEY = 'reject'
-      DEFAULT_REJECT_SET = [/^.$/, /[0-9]$/]
+      DEFAULT_REJECT_PATTERNS = [/^.$/, /[0-9]$/]
 
       # The name of the config field that lists the specific names that are
       # to be treated as exceptions; these names will not be reported as
       # uncommunicative.
       ACCEPT_KEY = 'accept'
-      DEFAULT_ACCEPT_SET = ['Inline::C']
+      DEFAULT_ACCEPT_PATTERNS = [/Inline::C/]
```
Comment (chastell) [inline lib/reek/smells/uncommunicative_module_name.rb]: So what about this default…? :)

Comment (troessner) [inline lib/reek/smells/uncommunicative_module_name.rb]: God damn it. Its so hot here my brain is fried.

**Comment by troessner** [comment]
Addressed all comment for realz this time, please re-review ;)

**Comment by chastell** [comment]
:+1:

File: features/samples.feature [hunk 2]
```diff
@@ -24,6 +24,7 @@ Feature: Basic smell detection
       Inline::C declares the class variable @@type_map (ClassVariable)
       Inline::C has at least 13 instance variables (TooManyInstanceVariables)
       Inline::C has at least 25 methods (TooManyMethods)
+      Inline::C has the name 'C' (UncommunicativeModuleName)
```
Comment (mvz) [inline features/samples.feature]: :+1: :smile:

**Pull request merged**